IoTCoreDefaultApp/IoTCoreDefaultApp/Views/OOBENetwork.xaml.cs

```diff
@@ -214,7 +214,7 @@ private void BackButton_Clicked(object sender, RoutedEventArgs e)
 
         private void SkipButton_Clicked(object sender, RoutedEventArgs e)
         {
-            CortanaHelper.LaunchCortanaToConsentPageAsyncIfNeeded();
+            CortanaHelper.LaunchCortanaToConsentPageAsyncIfNeeded()?.Wait();
```

Comment:
Remove the **Wait** and put the **await** keyword at the beginning of the line. Will need to add the **async** keyword before between **private** and **void**.